include/freebsd/osd.h

```diff
@@ -35,6 +35,7 @@
 
 #include <sys/endian.h>
 #include <pthread_np.h>
+#include <netinet/in.h>
```

Comment:
Hmm, yes. That might be a more standard compliant fix given that the header should exist on all platforms.